modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -33,7 +33,7 @@ To create and configure a GitHub App manually for {pac}, perform the following s
 
 * **GitHub Application Name**: `OpenShift Pipelines`
 * **Homepage URL**: OpenShift Console URL 
-* **Webhook URL**: The {pac} route or ingress URL. You can find it by executing the command `echo https://$(oc get route -n pipelines-as-code el-pipelines-as-code-interceptor -o jsonpath='{.spec.host}')`.
+* **Webhook URL**: The {pac} route or ingress URL. You can find it by executing the command `echo https://$(oc get route -n pipelines-as-code pipelines-as-code-controller -o jsonpath='{.spec.host}')`.
```

Comment:
Shouldn't we use `-n openshift-pipelines` here and in the note we ask user to use `pipelines-as-cod` (line number 40) as the customers who installed PAC using Openshift Pipelines operator will be using this doc?
@vdemeester @ppitonak

Reply:
yes please we need to update `-n pipelines-as-code` to `-n openshift-pipelines`